demo/common/src/lib.rs

```diff
@@ -81,14 +83,19 @@ mod device;
 mod renderer;
 mod ui;
 
+enum Content {
+    Svg(SvgTree),
+    Pdf(PdfFile<Vec<u8>>, PdfRenderCache, u32)
```

Comment:
I think this should probably be a struct-like enum variant, so that we can give the `u32` field a name.

Reply:
fixed